dmgroups.tex

```diff
@@ -83,6 +83,7 @@ \subsection{DM Systems Engineering Team \label{sect:sysengt}}
 \item Formalize the documentation tree for DM, defining which documents need to be produced for each product
 \item Agree the process for tracing the baseline requirements verification and validation status.
 \item Agree the formal versions of documents and software which form the technical baseline, individual items will go through the CCB for formal approval.  This includes upload to docushare.
+\item Perform releases of products such as pipelines software as needed using tooling provided by SQuARE (\secref{sect:square}.Debug any build problems related to the product (as opposed to the tooling).
```

Comment:
It feels a little bit weird to LDM this because it is such a special (and to some extent) weird case: why is pipelines "magic" in that pipeline jobs getting to be looked after by Arch, but all other product teams look after their own testing? But in any case, this seems to be the practice, so it is what it is.

Reply:
I'm doubly confused by this.

First, I didn't think that there was any expectation that folks other than Pipelines debugged Pipelines build problems. I mean, K-T and Tim do it compulsively because they can't help themselves, but I thought that was just them, rather than a requirement or expectation. I agree with Frossie that expecting somebody other than Pipelines to be responsible for debugging Pipelines is just... weird.

Secondly, this seems to be a description of the Systems Engineering Team, not the Architecture team. As far as I'm aware, the SET has never actually met (and I hope I'd have noticed, since I'm on it). Go figure.

Reply:
(Also, I think Frossie is misreading — the above says “products such as pipelines”, not “pipelines, but not other products.)

Reply:
ok -so I misunderstood what I was supposed to be clarifying .. anyone like ot offer me an alternative ?

Reply:
and yes Frossie you misread that

Reply:
So I'm actually not sure what the aim is here. :-)

I think that the product's developers (e.g., the Science Pipelines group in the case of the Pipelines) are responsible for fixing any build problems related to the product itself (ie, if we break afw, we have to fix afw), and that SQuaRE is responsible for fixing any build problems related to the build infrastructure (Jenkins, Travis, all that stuff).

I understood that to be the case at present, and I did not think it was changing.

Reply:
John clarified the product issue - i added a bullet on build infrastructure (as opposed to the release infrastructure). I am going to RFC - otherwise it will be review time already